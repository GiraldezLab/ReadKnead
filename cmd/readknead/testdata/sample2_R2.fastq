@HWI-ST1144:966:HJVL3ADXX:1:1101:1918:1964 2:N:0:AGTCAA
GTTCCACCTGTTCACCTTTCGTTGAAGAGCAAATGAAGAGGGAAAATGATAATGATAATATATTGCATCGTAATAG
+
@=@DDDADHDFBDGHGHHF>CFHIIIIGEGIGIG@?CFIGG<:?FHIHDCB0BFFFHIC<EBBFHDHGFAEGIC:D
@HWI-ST1144:966:HJVL3ADXX:1:1101:2813:1967 2:N:0:AGTCAA
AACTTCTGGATCGGTAGCCAAAGGAGAAATTGGGTCCCAGTGCAGCTGGTATTTCTGTTTGGCGAGATCGGAAGAG
+
@@CFFFFFHHHHHJJEHGGIIJIGHJIIFHHAFHIFIIHIIIJIIIDGGGIDFGEGDH?CCEICHIHGHHGHFH37
@HWI-ST1144:966:HJVL3ADXX:1:1101:3969:1976 2:N:0:AGTCAA
TCATGCTGACTTAAAAAAATCAAACTACACAGAATTGTCTTTTATAGTCAAACAATTTCTGTGGTGACGGTAAACT
+
@@@FDFFDDFHHHHIGEHBEH@GBGGHGHHCGGBEHF?FGGHGIIIE4BGIIBFGBAG@FA7@=)5@E=/?BFDEE
@HWI-ST1144:966:HJVL3ADXX:1:1101:5724:1968 2:N:0:AGTCAA
GACGGATCCAAACACAATGAAAAATGTGCTTCAATTTTACAGAATGGTGGCTCTCGAGGCCCAGATCGGAAGAGCA
+
@C@FFFFFHHDDHGJJJFGGGHGGJFFHEHIGJCHCAGFEHGCGGI@A6BFGGGECD@DGCHF>=AC9??DE7?B5
