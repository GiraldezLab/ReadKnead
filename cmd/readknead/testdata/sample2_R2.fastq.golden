@sample2.1
GTTCCACCTGTTCACCTTTCGTTGAAGAGCAAATGAAGAGGGAAAATGATAATGATAATATATTGCATCGTAATAG
+
@=@DDDADHDFBDGHGHHF>CFHIIIIGEGIGIG@?CFIGG<:?FHIHDCB0BFFFHIC<EBBFHDHGFAEGIC:D
@sample2.2
AACTTCTGGATCGGTAGCCAAAGGAGAAATTGGGTCCCAGTGCAGCTGGTATTTCTGTTTGGCG
+
@@CFFFFFHHHHHJJEHGGIIJIGHJIIFHHAFHIFIIHIIIJIIIDGGGIDFGEGDH?CCEIC
@sample2.3
TCATGCTGACTTAAAAAAATCAAACTACACAGAATTGTCTTTTATAGTCAAACAATTTCTGTGGTGACGGTAAACT
+
@@@FDFFDDFHHHHIGEHBEH@GBGGHGHHCGGBEHF?FGGHGIIIE4BGIIBFGBAG@FA7@=)5@E=/?BFDEE
@sample2.4
GACGGATCCAAACACAATGAAAAATGTGCTTCAATTTTACAGAATGGTGGCTCTCGAGGCCC
+
@C@FFFFFHHDDHGJJJFGGGHGGJFFHEHIGJCHCAGFEHGCGGI@A6BFGGGECD@DGCH
